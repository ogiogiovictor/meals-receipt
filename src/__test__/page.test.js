import React from 'react';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Category from '../pages/category/category';

const TestRenderer = require('react-test-renderer');

it('Renders meals correctly', () => {
  const tree = TestRenderer
    .create(
      <Provider store={store}>
        <Category />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
