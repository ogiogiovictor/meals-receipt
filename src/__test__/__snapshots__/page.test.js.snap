// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders meals correctly 1`] = `
<div
  className="meals-bg"
>
  <div
    className="category-header"
  >
    <h1>
      Categories
    </h1>
    <br />
    <hr />
    <table>
      <thead>
        <tr>
          <th>
            ID
          </th>
          <th>
            Category
          </th>
          <th>
            Image
          </th>
          <th>
            Description
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
